src/mousetracker.js

```diff
@@ -2879,6 +2879,12 @@
                 }
             }
 
+            // A primary mouse button may have been released while the non-primary button was down
+            if (pointsList.contacts > 0 && pointsList.type === 'mouse') {
+                // Stop tracking the mouse
+                pointsList.contacts--;
+                return true;
```

Comment:
I'm not terribly familiar with this code, so these questions are partially just for my own understanding:

* Why decrement the contacts rather than setting it to 0?
* Should it behave differently if contacts is 1 vs 2 or more?
* Why return true here?
* Should we be calling the release handler here if the contacts do get to 0?
* This looks like it covers the primary button being released while the non-primary is down... do we also need code for the other way?

Thank you for getting in here and learning about this stuff!